roborock/containers.py

```diff
@@ -629,6 +632,10 @@ def msg_ver(self) -> int:
     def msg_seq(self) -> int:
         return self.get(StatusField.MSG_SEQ)
 
+    @property
+    def status(self) -> str:
+        return STATE_CODE_TO_STATUS.get(self.state, f"Unknown Status {self.state}")
```

Comment:
I think this should be handled in another way. Returning a free text instead of the status itself looks bad. There are already attribute like state, error_code and the new ones you added handling the raw value

Reply:
What would you prefer? returning the state_code like before and just handling the get on the frontend like before? Perfectly fine with whatever you would like to do, just let me know.

Reply:
Yes. Just returning str mapping or None as before, and let whoever is using the data deal with it

Reply:
Okay, changed!